__tests__/utils/resizingConstraints.test.js

```diff
@@ -0,0 +1,35 @@
+const { calculateResizingConstraintValues, resizingConstraintValues } = require('../../utils/resizingConstraints.js');
+
+describe('Test sizing constraints', () => {
+  test('reduce series of numbers to their bitwise AND value', () => {
+    const { top, left, height } = resizingConstraintValues;
+    const input = [left, height];
+    // eslint-disable-next-line no-bitwise
```

Comment:
As you decide. I added local disable to pass the linter and to have under control the exceptions. It's not a good practice, but in this cases is needed because of Sketch logic.